src/sentry/static/sentry/app/views/discover/result/index.tsx

```diff
@@ -43,7 +43,7 @@ type ResultProps = {
   savedQuery: SavedQuery | null; // Provided if it's a saved search
   onFetchPage: (nextOrPrev: string) => void;
   onToggleEdit: () => void;
-  utc: boolean | null;
+  utc?: boolean;
```

Comment:
When I've touched utc in the past the null|undefined parts were tedious. You might need to keep null in the type to work with the inconsistent typing on this value elsewhere.